nomad/structs/structs.go

```diff
@@ -2514,6 +2514,17 @@ var (
 	}
 )
 
+var (
+	defaultServiceJobReschedulePolicy = ReschedulePolicy{
```

Comment:
It is useful to have these exported so you can use them in the api pkg when setting defaults